spring-cloud-dataflow-admin-local/pom.xml

```diff
@@ -34,8 +34,8 @@
 				<directory>src/main/resources</directory>
 				<filtering>true</filtering>
 				<includes>
-					<include>META-INF/spring.factories</include>
 					<include>admin.yml</include>
+					<include>META-INF/spring.factories</include>
```

Comment:
I think we need this entry in order to include `spring.factories` inside the package `jar`.